### ISSUE TITLE
WDS PPT Missing Alt Text

### ISSUE THREAD

**Issue by dzaremba90** [dzaremba90/MCW-Continuous-delivery-in-VSTS-and-Azure#2]
Hello, 

Please add missing alt text to the icons in the WDS trainer presentation - be sure to make changes to the ppt in the branch titled: MCW-Continuous-Delivery-in-VSTS-and-Azure_dz-edits.

Thank you, 

Diana
